### PR TITLE
Change image tagging for fluxv2

### PR DESCRIPTION
**Pull request by thomast1906**
Notes:
* Updates image tagging from `<imagetag>-<commit_id>` to `<imagetag>-<commit_id>-<last_commit_datetime>`
* Change required for fluxv2
* Changing to `commit?.substring(0, 7)` digits rather than `commit?.substring(0, 8)`

### CODE REVIEW

**Review by adusumillipraveen — APPROVED** [review]
LGTM, lets announce the change before/when we merge though as it breaks some customised pipelines.

File: src/uk/gov/hmcts/contino/DockerImage.groovy [hunk 1]
```diff
@@ -28,14 +28,16 @@ class DockerImage {
   Acr acr
   def commit
   def registryHost
+  def lastcommittime
 
 
-  DockerImage(product, component, acr, tag, commit) {
+  DockerImage(product, component, acr, tag, commit, lastcommittime) {
```
Comment (timja) [inline src/uk/gov/hmcts/contino/DockerImage.groovy:34]: camel case syntax please
```suggestion
  DockerImage(product, component, acr, tag, commit, lastCommitTime) {
```

normally we would make this compatible but doesn't look do-able unfortunately, let's see how it goes

File: src/uk/gov/hmcts/contino/DockerImage.groovy [hunk 2]
```diff
@@ -28,14 +28,16 @@ class DockerImage {
   Acr acr
   def commit
   def registryHost
+  def lastcommittime
 
 
-  DockerImage(product, component, acr, tag, commit) {
+  DockerImage(product, component, acr, tag, commit, lastCommitTime) {
```
Comment (timja) [inline src/uk/gov/hmcts/contino/DockerImage.groovy:34]: don't forget to update the other references

Comment (thomast1906) [reply src/uk/gov/hmcts/contino/DockerImage.groovy:34]: yup - sorted now, merged your recommendation rather than update all at once :D

**Review by adusumillipraveen — APPROVED** [review]

**Review by timja — APPROVED** [review]

**Pull request merged**